src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRMSqlDatabaseGeoBackup.md

```diff
@@ -138,4 +138,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Restore-AzureRmSqlDatabase](./Restore-AzureRmSqlDatabase.md)
 
-
+[SQL Database Documentation](https://docs.microsoft.com/azure/sql-database/)
```

Comment:
Because we're inside SQL DB cmdlets linking to SQL DB docs and staying within the context of Azure, it's ok without.